windows not installing static lib

Files in the current Windows package:

- Library\include\shogun\*
- Library\share\shogun\examples\libshogun\minimal.exe
- Library\lib\shogun.lib

@vigsterkr, should it be installing the static library too?

yep indeed... lemme check this error:
```
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\Microsoft.CppCommon.targets(171,5): error MSB6001: Invalid command line switch for "cmd.exe". The path is not of a legal form. [C:\bld\shogun_1511965094444\work\shogun-shogun_6.1.2\build\src\interfaces
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,8 +1,8 @@
 {% set name = "shogun-cpp" %}
 
-{% set version = "6.1.2" %}
+{% set version = "6.1.3" %}
 {% set commit = 'shogun_' + version %}
-{% set sha256 = "50cd1482d9977933b6ab4e410f4debc00bfa935fdf2812e41edb73504757c0df" %}
+{% set sha256 = "75f4d555efe06eaa7c4c12a1dc942f6e4d41a8ed495777a790b9bd9df936c19c" %}
 
 # Shogun doesn't produce a single tarball you can build from anymore;
 # need to get the gpl-shogun tarball too. Remember to update this when updating
@@ -77,16 +77,7 @@ build:
           -DUSE_SWIG_DIRECTORS=ON                      # [win]
     - if errorlevel 1 exit 1                           # [win]
 
-    - msbuild /p:Configuration=Release src\\shogun\\libshogun.vcxproj      # [win]
-    - if errorlevel 1 exit 1                                               # [win]
-
-    - msbuild /p:Configuration=Release src\\shogun\\shogun.vcxproj         # [win]
-    - if errorlevel 1 exit 1                                               # [win]
-
-    - msbuild /p:Configuration=Release src\\shogun\\shogun-static.vcxproj  # [win]
-    - if errorlevel 1 exit 1                                               # [win]
-
-    - msbuild /p:Configuration=Release shogun.sln        # [win]
+    - cmake --build . --target shogun --config Release   # [win]
     - if errorlevel 1 exit 1                             # [win]
 
     - cmake --build . --target install --config Release  # [win]
@@ -105,8 +96,7 @@ requirements:
     - cmake
     - ctags
     - curl >=7.44.0,<8
-    - eigen                     # [not win]
-    - eigen >=3.2,<3.3          # [win]
+    - eigen
     - glpk 4.61|4.62
     - hdf5 1.10.1
     - json-c 0.12|0.12.*        # [not win]
@@ -147,7 +137,7 @@ test:
 
     # linkage checks
     - python -c "import ctypes; ctypes.cdll['${PREFIX}/lib/libshogun${SHLIB_EXT}']"                      # [unix]
-    - python -c "import os, ctypes; ctypes.cdll[os.path.join(os.environ['LIBRARY_LIB'], 'shogun.lib')]"  # [win]
+    - python -c "import os, ctypes; ctypes.cdll[os.path.join(os.environ['LIBRARY_LIB'], 'shogun.dll')]"  # [win]
     - conda inspect linkages -p $PREFIX {{ name }}  # [not win]
     - conda inspect objects  -p $PREFIX {{ name }}  # [osx]
 

